migrations/2022-04-26-105145_only_mod_can_post/up.sql

```diff
@@ -0,0 +1 @@
+alter table community add column posting_restricted boolean default false;
```

Comment:
We need to be more explicit about what restricted means, even if the column name is really long.

Maybe `posting_restricted_to_mods`

Then the later ones can be `viewing_restricted_to_mods`, `commenting_restricted_to_mods`, etc.

Reply:
Updated. Though if we add more options like this, they will probably need their own db table.